internal/transform/operation.go

```diff
@@ -1600,11 +1601,13 @@ func contractIdFromContractData(ledgerKey xdr.LedgerKey) string {
 		return ""
 	}
 	contractIdHash, ok := contractData.Contract.GetContractId()
-	if ok {
-		return contractIdHash.HexString()
+	if !ok {
+		return ""
```

Comment:
> are there circumstances when we won't pull the contract id correctly for a contract invocation? 
There shouldn't be. Yes it's just to keep ingestion going in case something is wrong